requirements/constraints.txt

```diff
@@ -121,7 +121,7 @@ libsass==0.10.0
 click==8.1.6
 
 # pinning this version to avoid updates while the library is being developed
-openedx-learning==0.2.0
+openedx-learning==0.2.1
```

Comment:
Can we merge https://github.com/openedx/openedx-learning/pull/95 first and so change this to `openedx-learning==0.2.3`?